addons/account/models/chart_template.py

```diff
@@ -1046,6 +1057,17 @@ def _get_account_reconcile_model(self, template_code):
                     }),
                 ],
             },
+            'internal_transfer_reco': {
```

Comment:
From the specs, this should only be created if there are 2 similar journals.
So it should be created in the `create` of `account.journal`

Reply:
That is a nice to have in all fairness ; if that is too complicated, having an Internal Transfer by default is ok. We're doing that for l10n_be already.